diff --git a/pkg/virt-config/configuration.go b/pkg/virt-config/configuration.go
index 83aa81263a..c907f13dc5 100644
--- a/pkg/virt-config/configuration.go
+++ b/pkg/virt-config/configuration.go
@@ -260,6 +260,10 @@ func defaultClusterConfig(cpuArch string) *v1.KubeVirtConfiguration {
 				EmulatedMachines: strings.Split(DefaultAARCH64EmulatedMachines, ","),
 				MachineType:      DefaultAARCH64MachineType,
 			},
+			Ppc64le: &v1.ArchSpecificConfiguration{
+				EmulatedMachines: strings.Split(DefaultPPC64LEEmulatedMachines, ","),
+				MachineType:      DefaultPPC64LEMachineType,
+			},
 			S390x: &v1.ArchSpecificConfiguration{
 				OVMFPath:         DefaultS390xOVMFPath,
 				EmulatedMachines: strings.Split(DefaultS390XEmulatedMachines, ","),
