diff --git a/pkg/hypervisor/common/setsched.go b/pkg/hypervisor/common/setsched.go
index 29a7d02d24..f6b307e00f 100644
--- a/pkg/hypervisor/common/setsched.go
+++ b/pkg/hypervisor/common/setsched.go
@@ -1,4 +1,4 @@
-//go:build (linux && amd64) || (linux && arm64) || (linux && s390x)
+//go:build (linux && amd64) || (linux && arm64) || (linux && s390x) || (linux && ppc64le) || (linux && ppc64le)
 
 /*
  * This file is part of the KubeVirt project
