diff --git a/pkg/virt-config/virt-config.go b/pkg/virt-config/virt-config.go
index f63d9f54ff..9b347d580d 100644
--- a/pkg/virt-config/virt-config.go
+++ b/pkg/virt-config/virt-config.go
@@ -46,11 +46,13 @@ const (
 	DefaultAMD64MachineType                         = "q35"
 	DefaultAARCH64MachineType                       = "virt"
 	DefaultS390XMachineType                         = "s390-ccw-virtio"
+	DefaultPPC64LEMachineType                       = "pseries"
 	DefaultCPURequest                               = "100m"
 	DefaultMemoryOvercommit                         = 100
 	DefaultAMD64EmulatedMachines                    = "q35*,pc-q35*"
 	DefaultAARCH64EmulatedMachines                  = "virt*"
 	DefaultS390XEmulatedMachines                    = "s390-ccw-virtio*"
+	DefaultPPC64LEEmulatedMachines                  = "pseries*"
 	DefaultLessPVCSpaceToleration                   = 10
 	DefaultMinimumReservePVCBytes                   = 131072
 	DefaultNodeSelectors                            = ""
@@ -157,6 +159,8 @@ func (c *ClusterConfig) GetMachineType(arch string) string {
 		return c.GetConfig().ArchitectureConfiguration.Arm64.MachineType
 	case "s390x":
 		return c.GetConfig().ArchitectureConfiguration.S390x.MachineType
+	case "ppc64le":
+		return c.GetConfig().ArchitectureConfiguration.Ppc64le.MachineType
 	default:
 		return c.GetConfig().ArchitectureConfiguration.Amd64.MachineType
 	}
@@ -189,6 +193,8 @@ func (c *ClusterConfig) GetEmulatedMachines(arch string) []string {
 		return c.GetConfig().ArchitectureConfiguration.Arm64.EmulatedMachines
 	case "s390x":
 		return c.GetConfig().ArchitectureConfiguration.S390x.EmulatedMachines
+	case "ppc64le":
+		return c.GetConfig().ArchitectureConfiguration.Ppc64le.EmulatedMachines
 	default:
 		return c.GetConfig().ArchitectureConfiguration.Amd64.EmulatedMachines
 	}
