diff --git a/cmd/virt-launcher-monitor/virt-launcher-monitor.go b/cmd/virt-launcher-monitor/virt-launcher-monitor.go
index c3fef651b0..867ad02950 100644
--- a/cmd/virt-launcher-monitor/virt-launcher-monitor.go
+++ b/cmd/virt-launcher-monitor/virt-launcher-monitor.go
@@ -36,7 +36,6 @@ import (
 
 	"github.com/spf13/pflag"
 
-	"golang.org/x/sys/unix"
 	"kubevirt.io/client-go/log"
 )
 
@@ -101,9 +100,7 @@ func RunAndMonitor(containerDiskDir, uid string) (int, error) {
 	args := removeArg(os.Args[1:], "--keep-after-failure")
 
 	cmd := exec.Command("/usr/bin/virt-launcher", args...)
-	cmd.SysProcAttr = &syscall.SysProcAttr{
-		AmbientCaps: []uintptr{unix.CAP_NET_BIND_SERVICE},
-	}
+	cmd.SysProcAttr = &syscall.SysProcAttr{}
 	cmd.Stdout = os.Stdout
 	cmd.Stderr = os.Stderr
 
